src/GovUk.Education.ExploreEducationStatistics.Notifier/PublicationNotifier.cs

```diff
@@ -123,8 +121,6 @@ public class PublicationNotifier
             slug = slug ?? data?.slug;
             title = title ?? data?.title;
 
-            logger.LogInformation($"email: {email} id: {id} slug: {slug} title: {title}");
```

Comment:
Lol nice!

Reply:
Also RowId being logged elsewhere is the email address.